### ISSUE TITLE
1.3.48 Specific livestreams getting stuck

### ISSUE THREAD

**Issue by aikaterna** [sedmelluq/lavaplayer#481]
Specific YouTube livestreams return a track stuck after a few moments. The only example url I have currently is https://www.youtube.com/watch?v=KVbHBto_lwk which is an event that is being broadcasted live as this issue is being composed (not sure if the track will continue to be stuck after the events are over). Other yt livestreams like lofi radio (https://www.youtube.com/watch?v=5qap5aO4i9A) and NCS (https://www.youtube.com/watch?v=Oxj2EAr256Y) still work as expected.

I am a Lavalink user and the info about the track displayed is not very helpful as there are no exceptions shown.

```
2020-05-18 04:10:10.372  INFO 12408 --- [   XNIO-1 I/O-2] lavalink.server.io.SocketServer          : {"op": "play", "guildId": "133049272517001216", "track": "QAAAhAIAIUVEQyBMYXMgVmVnYXMgVmlydHVhbCBSYXZlLUEtVGhvbgAJSW5zb21uaWFjf/////////8AC0tWYkhCdG9fbHdrAQEAK2h0dHBzOi8vd3d3LnlvdXR1YmUuY29tL3dhdGNoP3Y9S1ZiSEJ0b19sd2sAB3lvdXR1YmUAAAAAAAAAAA=="}
2020-05-18 04:10:20.404  WARN 12408 --- [ntLoopGroup-2-4] lavalink.server.player.EventEmitter      : EDC Las Vegas Virtual Rave-A-Thon got stuck! Threshold surpassed: 10000
```

**Comment by aikaterna** [comment]
Another active livestream at this time that triggers this error: https://www.youtube.com/watch?v=ZTAcNWEZRzI
I am currently using Lavaplayer 1.3.49 in my Lavalink.jar.

**Comment by realmayus** [comment]
Can confirm: https://www.youtube.com/watch?v=5yx6BWlEVcY

**Comment by nikammerlaan** [comment]
I did a lot of digging with this and found that this happens when YouTube returns the mpeg file with no sidx header. This causes it to be parsed with a different parser which doesn't take kindly to it.

**Comment by GrieVeR-13** [comment]
Also found that the problem is sidx header. All day I tried to set up a stream for YouTube, so that he returns the headers, but to no avail. Any ideas?

**Comment by DeinAlbtraumTV** [comment]
This still isn´t fixed, even in version 1.3.59. Do you Guys have found any workarounds/fixes?

**Comment by DeinAlbtraumTV** [comment]
> I did a lot of digging with this and found that this happens when YouTube returns the mpeg file with no sidx header. This causes it to be parsed with a different parser which doesn't take kindly to it.

Wouldn´t it be possible to manually set the Header before passing the request on to the parser?

**Comment by aikaterna** [comment]
This is fixed with Lavaplayer 1.3.56. Thanks sed ❤️
